src/main/webapp/WEB-INF/jsp/contributions/donate.jsp

```diff
@@ -247,4 +247,26 @@
             </script>
         </div>
     </div>
+
+    <!-- Get donations list using Covalent API -->
+    <script type="text/javascript">
+        var url = 'https://api.covalenthq.com/v1/1/address/${applicationScope.configProperties['donationAddress']}/transactions_v2/?no-logs=true&limit=10&match=%7B%22to_address%22%3A+%22${applicationScope.configProperties['donationAddress']}%22%2C+%22value%22%3A+%7B+%22%24gt%22%3A+0+%7D++%7D&key=${applicationScope.configProperties['covalentApiKey']}';
```

Comment:
Does Covalent also support the Rinkeby testnet?

Reply:
Unfortunately it doesn't support Rinkeby but it does support Kovan (https://www.covalenthq.com/docs/networks#testnets).

I can create a test setup with Kovan if you like.

Reply:
> Unfortunately it doesn't support Rinkeby but it does support Kovan (https://www.covalenthq.com/docs/networks#testnets).
> 
> I can create a test setup with Kovan if you like.

I see. Then let's stick with the current solution (Ethereum Mainnet in both `TEST` and `PROD`).